migrations/20190321234838_create_votes_table.js

```diff
@@ -0,0 +1,35 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('votes', function(table) {
+    table.increments('vote_id');
+    table.integer('user_id').notNullable();
+    table.integer('meme_id').notNullable();
+    table.enu('type', ['up', 'down']).notNullable();
+    table.timestamp('date_created').defaultTo(knex.fn.now());
+
+    table.unique(['user_id', 'meme_id', 'type']);
```

Comment:
Drop `type` from this constraint. As is, I could vote once up and another time down.